zeek/compiler/glue-compiler.cc

```diff
@@ -198,11 +198,14 @@ static hilti::rt::Port extract_port(const std::string& chunk, size_t* i) {
         throw ParseError("cannot parse port specification");
 
     hilti::rt::Protocol proto;
-    uint16_t port;
+    uint64_t port;
```

Comment:
Line 194 will already lead to us rejecting negative numbers (e.g., `-2/udp`), but it would be great to add a test for that as well so we do not accidentally introduce changes causing well-defined, but unintended unsigned integer overflow here.